File: GlassDoor/ClientApp/src/app/models/education-level.ts
import { Employee } from "./employee";
import { JobDetails } from "./job-details";

export class EducationLevel {

  constructor(id?: number, name?: string, jobDetails?: JobDetails,
    employees?: Employee) {

    this.id = id;
    this.name = name;
    this.jobDetails = jobDetails;
    this.employees = employees;

  }
  public id: number;
  public name: string;
  public employees: Employee;
  public jobDetails: JobDetails;
}

File: DAL/Models/EducationLevel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DAL.Models
{
    public class EducationLevel
    {
        public int Id { get; set; }
        public string Name { get; set; }

        public ICollection<JobDetails> JobDetails { get; set; }
        public ICollection<Employee> Employees { get; set; }
    }
}
